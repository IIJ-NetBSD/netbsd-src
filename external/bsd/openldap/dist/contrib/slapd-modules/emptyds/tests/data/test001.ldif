# slapd prevents us from adding the same value multiple times
dn: cn=Ursula Hampster,ou=Alumni Association,ou=People,dc=example,dc=com
changetype: modify
add: description
description: one
description: 
description: two
description: three
description: four
# a space is distinct from an empty value
description:: ICAg
-
replace: drink
drink: Earl Grey, hot
-
delete: description
description: 
-
replace: drink
drink: Earl Grey, hot

# there is no such restriction on deletes, so we exercise this part of the overlay here
dn: cn=Ursula Hampster,ou=Alumni Association,ou=People,dc=example,dc=com
changetype: modify
delete: description
description: 
description: four
description: 
description: three
description: two
description: 
description: 
description: one
description: 
-
add: description
description: 

dn: cn=James A Jones 1,ou=Alumni Association,ou=People,dc=example,dc=com
changetype: modify
replace: drink
drink: 

dn: cn=All Staff,ou=Groups,dc=example,dc=com
changetype: modify
delete: member
-
add: member
# an empty DN should not be stripped
member: 
member: cn=Dorothy Stevens,ou=Alumni Association,ou=People,dc=example,dc=com

dn: cn=Gern Jensen,ou=Information Technology Division,ou=People,dc=example,dc=com
changetype: add
objectclass: testPerson
cn: Gern Jensen
sn: Jensen
uid: gjensen
title: 
postaladdress: ITD $ 535 W. William St $ Anytown, MI 48103
seealso: cn=All Staff,ou=Groups,dc=example,dc=com
drink: Coffee
homepostaladdress: 844 Brown St. Apt. 4 $ Anytown, MI 48104
description: Very odd
description: 
description: More than you think
facsimiletelephonenumber: +1 313 555 7557
telephonenumber: +1 313 555 8343
mail: gjensen@mailgw.example.com
homephone: +1 313 555 8844
testTime: 20050304001801.234Z
