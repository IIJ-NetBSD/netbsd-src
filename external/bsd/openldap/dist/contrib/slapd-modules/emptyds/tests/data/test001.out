dn: reqStart=timestamp,cn=log
reqDN: cn=Ursula Hampster,ou=Alumni Association,ou=People,dc=example,dc=com
reqMod: description:+ one
reqMod: description:+ two
reqMod: description:+ three
reqMod: description:+ four
# "description:+    " that's a space, then 3 spaces for value
reqMod:: ZGVzY3JpcHRpb246KyAgICA=
reqMod: drink:= Earl Grey, hot
# second mod was removed, so we have two replaces in succession now and need
# to separate them (":")
reqMod:: Og==
reqMod: drink:= Earl Grey, hot

dn: reqStart=timestamp,cn=log
reqDN: cn=Ursula Hampster,ou=Alumni Association,ou=People,dc=example,dc=com
reqMod: description:- four
reqMod: description:- three
reqMod: description:- two
reqMod: description:- one
# second mod is removed

dn: reqStart=timestamp,cn=log
reqDN: cn=James A Jones 1,ou=Alumni Association,ou=People,dc=example,dc=com
reqMod: drink:=

dn: reqStart=timestamp,cn=log
reqDN: cn=All Staff,ou=Groups,dc=example,dc=com
reqMod: member:-
# "member:+ " adding an empty DN
reqMod:: bWVtYmVyOisg
reqMod: member:+ cn=Dorothy Stevens,ou=Alumni Association,ou=People,dc=example
 ,dc=com

dn: reqStart=timestamp,cn=log
reqDN: cn=Gern Jensen,ou=Information Technology Division,ou=People,dc=example,
 dc=com
reqMod: objectClass:+ testPerson
reqMod: cn:+ Gern Jensen
reqMod: sn:+ Jensen
reqMod: uid:+ gjensen
reqMod: postalAddress:+ ITD $ 535 W. William St $ Anytown, MI 48103
reqMod: seeAlso:+ cn=All Staff,ou=Groups,dc=example,dc=com
reqMod: drink:+ Coffee
reqMod: homePostalAddress:+ 844 Brown St. Apt. 4 $ Anytown, MI 48104
reqMod: description:+ Very odd
reqMod: description:+ More than you think
reqMod: facsimileTelephoneNumber:+ +1 313 555 7557
reqMod: telephoneNumber:+ +1 313 555 8343
reqMod: mail:+ gjensen@mailgw.example.com
reqMod: homePhone:+ +1 313 555 8844
reqMod: testTime:+ 20050304001801.234Z
reqMod: structuralObjectClass:+ testPerson

